docs/usage.md

```diff
@@ -171,3 +171,7 @@ Rust is one of the many output languages we support. Check out this [basic examp
 ## Generate Python models
 
 Python is one of the many output languages we support. Check out this [basic example for a live demonstration](../examples/generate-python-models) and the following [Python documentation for more advanced use-cases](./languages/Python.md).
+
+## Generate Kotlin models
+
+Kotlin is one of the many output languages we support. Check out this [basic example for a live demonstration](../examples/generate-kotlin-models) and the following [Kotlin documentation for more advanced use-cases](./languages/Kotlin.md).
```

Comment:
Maybe it makes sense to link to both the `generate-kotlin-models` and the `generate-kotlin-enums` here 🤔